openedx/core/djangoapps/waffle_utils/views.py

```diff
@@ -76,6 +94,72 @@ def _add_waffle_flag_state(self, flags_dict):
             flag['created'] = str(flag_data.created)
             flag['modified'] = str(flag_data.modified)
 
+    def _add_waffle_flag_course_override_state(self, flags_dict):
```

Comment:
**Reviewer Note**: This is complicated, but may change completely soon.  If you spot a bug or simple clean-up, let me know.  Otherwise, let's get the prototype merged.